devsetup/Makefile

```diff
@@ -8,7 +8,7 @@ EDPM_TOTAL_NODES ?= 1
 BMAAS_INSTANCE_NAME_PREFIX ?= crc-bmaas
 BMAAS_INSTANCE_MEMORY ?= 4096
 BMAAS_INSTANCE_VCPUS ?= 2
-BMAAS_INSTANCE_DISK_SIZE ?= 10
+BMAAS_INSTANCE_DISK_SIZE ?= 20
```

Comment:
why do you need this change for kuttl tests? this seems to be used only for BMAAS scenarios.

Reply:
Thanks, resolved.